projects/plugins/jetpack/extensions/blocks/cookie-consent/block.json

```diff
@@ -0,0 +1,69 @@
+{
+	"$schema": "https://schemas.wp.org/trunk/block.json",
+	"apiVersion": 3,
+	"name": "jetpack/cookie-consent",
+	"title": "Cookie Consent",
+	"description": "Displays a customizable cookie consent banner. To display this block on all pages of your site, please add it inside a Template Part that is present on all your templates, like a Header or a Footer.",
```

Comment:
Not for this PR, but this makes me realize that we do not use consistent phrasing in our block descriptions. Sometimes we use third-person like here, while in other descriptions we typically use second person. Maybe in a follow-up PR we can update all blocks to use a consistent messaging.